addon/chrome/content/tab/tabBindings.js

```diff
@@ -224,7 +224,7 @@
       if (aTab.mIsHover && aTab.getAttribute("showbutton") != "on" &&
         !aTab.hasAttribute("dragged")) {
         if (TabmixTabbar.widthFitTitle)
-          aTab.style.setProperty("width", Tabmix.getBoundsWithoutFlushing(aTab).width + "px", "important");
+          aTab.style.setProperty("max-width", Tabmix.getBoundsWithoutFlushing(aTab).width + "px", "important");
```

Comment:
Why max-width ?
everywhere else i did use ```style.setProperty("width"....```

Reply:
I've tested 'width' has no effect here anymore tab width changes regardless.

Reply:
I have to test it more, i'v notice that that tab width preference not working properly.
if it true that we need to change it everywhere in the code that handle tab width when TabmixTabbar.widthFitTitle is true
lets move this issue to different PR.

just remove this change from here

Reply:
Ok, it looks like this is not the real problem I have some styles relate to width fit title left not been fixed, once fixed those 'width' seems functioning fine.